doc/modules/homology.rst

```diff
@@ -14,6 +14,7 @@ Undirected simplicial homology
    :template: class.rst
 
    homology.VietorisRipsPersistence
+   homology.WeightedRipsPersistence
```

Comment:
I think it should be added in `gtda/homology/__init__.py` as well

Reply:
Thanks! Of course 😊